bugout/calls.py

```diff
@@ -1,9 +1,11 @@
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict
 
 import requests
 
 from .data import Method
 
+REQUESTS_TIMEOUT = 5
```

Comment:
This should be `os.environ.get("BUGOUT_TIMEOUT_SECONDS", "5")`.

Reply:
Done